ChangeLog

```diff
@@ -1,3 +1,15 @@
+Changes in 2.2.3 Release on 2019-05-31:
```

Comment:
If you want to update the ChanegLog yourself (usually, I do it when I make a release), please use 20XX-XX-XX for the as-of-yet unreleased date.

Reply:
Corrected release date text in ChangeLog to be "20XX-XX-XX".